src/io/files.js

```diff
@@ -869,7 +877,17 @@ p5.prototype.httpDo = function () {
       if (typeof a === 'string') {
         if (a === 'GET' || a === 'POST' || a === 'PUT' || a === 'DELETE') {
           method = a;
-        } else if(a === 'json' || a === 'jsonp' || a === 'xml' || a === 'text') {
```

Comment:
I removed the or || because they were breaking the code style guides, otherwise would have left as is